docs/05-internals/development/bazel.md

```diff
@@ -4,21 +4,27 @@ The following section discusses on how to build and run tests for Knora-API
 with [Bazel](https://bazel.build).
 
 ## Prerequisites
-To install Bazel on macOS, run `brew install bazel`.
+To install the Bazel build tool, follow these steps:
+
+```
+$ npm install -g @bazel/bazelisk
+```
+
+This will install [bazelisk](https://github.com/bazelbuild/bazelisk) which is
+a wrapper to the `bazel` binary. It will, when the `bazel` command ir run,
+automatically install the supported Bazel version, defined in the `.bazelversion`
+file in the root of the `knora-api` repository.
 
 ## Commands
 
 Build `webapi`:
 
 ```
 # build webapi
-$ bazel build //webapi
-
-# run api (webapi)
-$ bazel run //:api
```

Comment:
so there is no `bazel run` command to run the api anymore ? 
I remember there was a `bazel run \\...` to run everything, do you wanna add it to documentation?

Reply:
There is. Just running the API does not make much sense without the whole stack. It won't work.

Reply:
With `bazel` you can only run one target at a time. This is why we still have `make` and `docker-compose`.